base/rational.jl

```diff
@@ -105,7 +105,24 @@ function //(x::Rational, y::Rational)
 end
 
 //(x::Complex, y::Real) = complex(real(x)//y, imag(x)//y)
-//(x::Number, y::Complex) = x*conj(y)//abs2(y)
+function //(x::Number, y::Complex)
+    if isinf(y)
+        return complex(x*0)
```

Comment:
This is type unstable.

You could safely fall back to regular `/` once #53453 is merged.

Reply:
newer version shouldn't have any instability